src/modelfree/shaping_wrappers.py

```diff
@@ -49,20 +51,31 @@ def log_callback(self, logger):
         for rew_type, rews in self.ep_rew_dict.items():
             assert len(rews) == num_episodes
             means[rew_type] = sum(rews) / num_episodes
-            logger.logkv(f'ep{rew_type}mean', means[rew_type])
+            logger.logkv(f'shaping/ep{rew_type}mean', means[rew_type])
 
         overall_mean = _anneal(means, self.reward_annealer)
-        logger.logkv('eprewmean_true', overall_mean)
+        logger.logkv('shaping/eprewmean_true', overall_mean)
+        if self.reward_annealer is not None:
+            c = self.reward_annealer()
+            logger.logkv('shaping/rew_anneal_c', c)
 
         for rew_type in self.ep_rew_dict:
             self.ep_rew_dict[rew_type] = []
 
+    def get_log_buffer_data(self):
+        """Return data to be analyzed by a ConditionalAnnealer"""
```

Comment:
Describe what the function does, e.g. what data is returned, not just it's intended use case.